### ISSUE TITLE
High GPU Usage

### ISSUE THREAD

**Issue by wpmichel** [CookieMonsterTeam/CookieMonster#920]
I have noticed that when I run the addon, my 3070 gets pinned to max and throttles other applications using the GPU. 
Is this addon supposed to be hardware accelerated? If so, can I configure it to be CPU bound to free up my GPU? 

I am using FireFox

![image](https://user-images.githubusercontent.com/42211647/127413757-108c50b3-e31d-47d4-902a-24fda880b817.png)

**Comment by DanielNoord** [comment]
Oh wow. Is this only with the add-on on? Or also with base cookie clicker?

I have no idea why this would be the case. I have no idea how to start hardware acceleration from javascript (or any language for that matter). Perhaps others have an idea about why this might be happening?

**Comment by wpmichel** [comment]
This is with both running, through I dont see the spike until loading the addon. 

I tried turning off FireFox hardware acceleration and that helped (though minorly) 

I didnt think that browsers would have access to GPUs, and I cant imagine that the render requirement is that high. 

Perhaps its related to that Javascript 'Game' object referenced in the helper bookmark?

**Comment by DanielNoord** [comment]
The `Game` object is created by cookie clicker itself. It basically contains all code of the game.

Do you know how to do profiler snapshot? Perhaps that shows a function taking considerable time on your system.
Another (recurring) problem is the memory usage of CM, but I don't know if that would affect GPU acceleration?

**Comment by wpmichel** [comment]
Im not familiar with profiler snapshot. Can you point me to it? Is part of the developer tooling?

**Comment by DanielNoord** [comment]
Yes! Open the developer tooling within Chrome (or any Chromium based browser), click on Performance, click on Record (the red button), click some things within the site, click stop recording and click save recording.

See also [here](https://developer.chrome.com/docs/devtools/evaluate-performance/)
